APMrover2/defines.h

```diff
@@ -90,9 +90,13 @@ enum mode {
 #define MASK_LOG_STEERING  		(1<<13)
 #define MASK_LOG_RC     		(1<<14)
 #define MASK_LOG_ARM_DISARM     (1<<15)
-#define MASK_LOG_WHEN_DISARMED  (1UL<<16)
 #define MASK_LOG_IMU_RAW        (1UL<<19)
 
+// Log behaviour values
+#define LOG_WHEN_DISARMED               0
```

Comment:
This param (and enum) belongs in DataFlash.cpp where LOG_BACKEND_TYPE and LOG_FILE_BUFSIZE live.